app/services/proofing/mock/state_id_mock_client.rb

```diff
@@ -25,6 +25,12 @@ class StateIdMockClient < Proofing::Base
       TRANSACTION_ID = 'state-id-mock-transaction-id-456'
 
       proof do |applicant, result|
+        if IdentityConfig.store.simulate_mva_timeout
+          raise ::Proofing::TimeoutError.new(
+            'ExceptionId: 0047, ExceptionText: MVA did not respond in a timely fashion',
+          )
+        end
```

Comment:
Could we trigger this via a specific drivers license number or something instead, similar to our phone number values? https://developers.login.gov/testing/#phone-number-verification

Reply:
Agree - made it recognize a last name of MVATimeout in bb80864